packages/blockchain-utils-linking/src/index.ts

```diff
@@ -5,3 +5,4 @@ export * as filecoin from "./filecoin";
 export * as polkadot from "./polkadot";
 export * as eosio from "./eosio";
 export * as cosmos from "./cosmos";
+export * as near from "./near";
```

Comment:
Can you also export the `NearAuthProvider` directly? See: https://github.com/ceramicnetwork/js-ceramic/pull/882

Reply:
yep, sounds good